### PR TITLE
MNT: Re-rendered with conda-smithy 1.4.4

### PR DESCRIPTION
**Pull request by frol**

### CODE REVIEW

**Comment by frol** [comment]
@jakirkham I have just noticed that the re-rendered scripts target only Python 3.5 packages. Is this the expected behavior?

File: .travis.yml [hunk 1]
```diff
@@ -7,11 +7,6 @@ os: osx
 osx_image: beta-xcode6.1
 
 env:
-  matrix:
-    
-    - CONDA_PY=27
-    - CONDA_PY=34
-    - CONDA_PY=35
```
Comment (jakirkham) [inline .travis.yml]: Ack! This doesn't look right. Could you please revert this @frol and raise an issue at [conda-smithy]( https://github.com/conda-forge/conda-smithy )?

Comment (jakirkham) [inline .travis.yml]: Actually just reopened issue ( https://github.com/conda-forge/conda-smithy/issues/223 ) instead. Please just revert for now.

**Comment by jakirkham** [comment]
> I have just noticed that the re-rendered scripts target only Python 3.5 packages. Is this the expected behavior?

I'm not totally sure what you are referring to, but it may be the problem pointed out above. If you see things like the Python matrix being dropped, this is a bug. We thought we had a sufficient workaround, but perhaps we do not. The underlying cause remains unknown. Please issue ( https://github.com/conda-forge/conda-smithy/issues/223 ) for some history.

**Pull request merged**